lib/builders/testArgBuilder.js

```diff
@@ -0,0 +1,40 @@
+module.exports = (value) => {
+    let args = [];
+    if(!value){
```

Comment:
Inconsistent spacing in js files for if statement "if()" vs "if ()". There are some other formatting inconsistencies in the js files. Run a code formatter on all files.